### ISSUE TITLE
What the form of timestamp used in time image

### ISSUE THREAD

**Issue by doublemanyu** [better-flow/better-flow#8]
Hi, there.Thanks for your work.
1)  I have a question about the timestamp used in time image T. Is the timestamp like 1e9 or 1e-9?
2) If the timestamp like 1e9, follow formula(9), why the warped coordination was computed in the range of (240,180)?
Thank you in advance.

**Comment by ncos** [comment]
Hi @doublemanyu! Could you please give me a bit more context for your question? What do you mean by 'like 1e9 or 1e-9'? Could you point me specifically to the code you are asking about?

1. The purpose of the timestamp image is to compute the gradient on it; i.e. it is the derivative of the timestamps what matters, not the absolute values. If I remember correctly, the image was constructed using relative timestamps (<current t> - <time window start>), so timestamps would not be 1e9. I do not think they would be as small as 1e-9 as well - typical values should be 10-100ms.
2. The resolution of the event camera (DVS, older version) is (240,180) pixels, so the resolution of the time image is too.

**Comment by doublemanyu** [comment]
@ncos Hi, there. Thanks for your reply.
What makes me is that when I compute the time image and visualize it.
In EED dataset, the timestamp like "1519939778.722000347", while in EVIMO dataset the timestamp like"0.016724356".  If the image was constructed using relative timestamps, I got it.
The second question is, what does the t means in the formula(10)? Is it relative timestamp of originally coordination(x,y,t)?

**Comment by ncos** [comment]
Yes, the model is applied in a small time window, so t represents a relative time in all formulas.

**Comment by doublemanyu** [comment]
> Yes, the model is applied in a small time window, so t represents a relative time in all formulas.

Hi, there.
What does "| ****|" mean in formula(10)? Does "| |" mean the detValue of array?
